docs/tutorials/craft-a-snap.rst

```diff
@@ -80,12 +80,12 @@ the build environment.
 
 .. literalinclude:: code/craft-a-snap/task.yaml
     :language: bash
-    :start-at: snap install --channel 5.21/stable lxd
```

Comment:
@mr-cal I should have confirmed with you. I have the Docker conflict, so I don't test LXD as often. Is 5.21 still a requirement?

Reply:
Your change is fine.  LXD reworked their release process, so now `snap install lxd` installs the latest LTS version.